### ISSUE TITLE
add validation set

### ISSUE THREAD

**Issue by EmbraceLife** [happynoom/DeepTrade_keras#3]
Hi @happynoom 

you have been using test set as validation set in training. 

How about making training set, validation set and test set instead of just training set and test set( which is used as both validation and test set)?  It should help to see whether model is really robust or not.

**Comment by happynoom** [comment]
OK，I will add more dataset to test this model. Maybe in this weekend.
